src/orion/algo/space.py

```diff
@@ -621,6 +627,46 @@ def get_category(value):
         return casted_point
 
 
+class Fidelity(Dimension):
+    """Subclass of `Dimension` for representing real parameters.
```

Comment:
```suggestion
    """Fidelity `Dimension` for multi-fidelity.
    
    Fidelity dimensions are not optimized by the algorithms. If it supports multi-fidelity, the algorithm will select a fidelity
    level for which it will sample hyper-parameter values to explore a low fidelity space. This class is used as a 
    place-holder so that algorithms can discern fidelity dimensions from hyper-parameter dimensions.
```

Reply:
Part of the doc is missing from your commit